curricula/models.py

```diff
@@ -62,6 +64,10 @@ class Curriculum(InternationalizablePage, RichText, CloneableMixin):
     class Meta:
         verbose_name_plural = "curricula"
 
+    @classmethod
+    def internationalizable_fields(cls):
+        return super(Curriculum, cls).internationalizable_fields() + ['assessment_commentary']
+
```

Comment:
@Hamms is this the correct way to make this field internationalizable?

Reply:
Yep!